tchannel-core/src/test/java/com/uber/tchannel/tracing/AsyncTracingContextTest.java

```diff
@@ -1,132 +1,40 @@
+/*
```

Comment:
main tests functionality was extracted to shared `AsyncTracingContextTestBase` parent class.